Documentation/Usage.md

```diff
@@ -1,27 +1,27 @@
 # Usage
 
-## Table of Contents
+## Table of contents
```

Comment:
Why change all the headings?

Reply:
the capitalization was inconsistent so i changed them all to sentence case